modules/ROOT/partials/configuration/init_content_sync.adoc

```diff
@@ -0,0 +1,22 @@
+[[init_content_sync]]
+== `+init_content_sync+`
+
+This option enables you to change the way the iframe is initialized from using srcdoc and being async to using document.write and being sync. Setting
+this to `true` will reduce some initialization flicker. But some flicker like loading the content css in could still occur.
```

Comment:
```suggestion
This option determines whether the iframe is initialized synchronously or asynchronously.

When disabled, the editor is initialized asynchronously using `srcdoc`, however this can occasionally cause the editor to flicker during initialization.
When enabled, the editor will be initialized using `document.write`, which can reduce this flickering, but some flicker could still occur, such as loading the content css.
```
Mainly wanted to move the word "Setting" onto the correct line, but thought I'd attempt a rewrite.

Reply:
That is so much better. Thanks a ton.